scripts/generators/vulkan_object.py

```diff
@@ -1,10 +1,19 @@
 #!/usr/bin/python3 -i
```

Comment:
so sorry I haven't got to a point where this is all unified in put the `Vulkan-Headers` yet, but we can have a seperate PR where we unify `vulkan_object.py` and `base_generator.py` across everywhere